ext/mysqli/tests/bind_insert/datetime_types.phpt

```diff
@@ -1,43 +1,43 @@
 --TEST--
-mysqli bind_param/bind_result date
+mysqli insert (bind_param + bind_result) datetime types
 --EXTENSIONS--
 mysqli
 --SKIPIF--
 <?php
-require_once('skipifconnectfailure.inc');
+require_once dirname(__DIR__) . "/test_setup/test_helpers.inc";
+mysqli_check_skip_test();
 ?>
 --FILE--
 <?php
-    require_once("connect.inc");
+require_once dirname(__DIR__) . "/test_setup/test_helpers.inc";
 
-    /*** test mysqli_connect 127.0.0.1 ***/
-    $link = my_mysqli_connect($host, $user, $passwd, $db, $port, $socket);
+    $link = default_mysqli_connect();
 
-    mysqli_select_db($link, $db);
+    mysqli_select_db($link, get_default_database());
     mysqli_query($link, "SET sql_mode=''");
```

Comment:
@kamil-tekiela are those two lines actually necessary? As we are pretty inconsistent about using them and wonder if we could just simplify the tests by removing these lines?

Reply:
`mysqli_select_db($link, $db);` is unnecessary because it's already done in line above it: `$link = my_mysqli_connect($host, $user, $passwd, $db, $port, $socket);`

`mysqli_query($link, "SET sql_mode=''");` is needed because of the test assumption: `%s(19) "0000-00-00 00:00:00"`. Without we would be getting inconsistent test results depending on the DB version/setup.